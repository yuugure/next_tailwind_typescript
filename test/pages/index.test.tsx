import Index from '../../pages/index'
import { render } from '../testUtils'

describe('Index page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Index />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
