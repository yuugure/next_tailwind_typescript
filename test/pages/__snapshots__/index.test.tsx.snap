// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page matches snapshot 1`] = `
<DocumentFragment>
  <header>
    <nav
      class="flex items-center justify-between p-4 flex-no-wrap sm:flex-wrap bg-gray-300"
    >
      <ul>
        <li
          class="mr-6"
        >
          <a
            class="text-white hover:text-gray-300"
            href="/"
          >
            Home
          </a>
        </li>
      </ul>
    </nav>
  </header>
  <main
    role="main"
  >
    <div
      class="container mx-auto"
    >
      <div
        class="flex justify-center mt-10"
      >
        <div
          class="flex flex-col items-center"
        >
          <img
            alt="私のアバター"
            class="rounded-full h-32 w-32"
            src="/images/myprofile.png"
          />
          <h1
            class="text-gray-700 text-4xl my-4"
          >
            Yuuichi Nishioka
          </h1>
          <p
            class="text-gray-600 text-justify px-8 mb-8"
          >
            Webエンジニア
          </p>
        </div>
      </div>
      <div
        class="flex flex-wrap mt-10 text-gray-600"
      >
        <div
          class="mb-6 p-4 md:w-1/3 md:mb-0 flex flex-col justify-center items-center max-w-sm mx-auto"
        >
          <div
            class="w-70 bg-white -mt-10 shadow-lg rounded-lg overflow-hidden p-5"
          >
            <div
              class="inline-flex"
            >
              <div
                class="flex-1 h-4 w-4 m rounded-full m-1 bg-yellow-100"
              >
                <div
                  class="h-2 w-2 rounded-full m-1 bg-yellow-500"
                />
              </div>
              <div
                class="text-gray-700"
              >
                趣味
              </div>
            </div>
            <div
              class="text-gray-600 text-left"
            >
              自宅にいるときは、映画・アニメ鑑賞・料理を作ったりしています。休日はキャンプ・海で釣り・ドライブをしています。
            </div>
          </div>
        </div>
        <div
          class="mb-6 p-4 md:w-1/3 md:mb-0 flex flex-col justify-center items-center max-w-sm mx-auto"
        >
          <div
            class="w-70 bg-white -mt-10 shadow-lg rounded-lg overflow-hidden p-5"
          >
            <div
              class="inline-flex"
            >
              <div
                class="flex-1 h-4 w-4 m rounded-full m-1 bg-yellow-100"
              >
                <div
                  class="h-2 w-2 rounded-full m-1 bg-yellow-500"
                />
              </div>
              <div
                class="text-gray-700"
              >
                特技
              </div>
            </div>
            <div
              class="text-gray-600 text-left"
            >
              Web系のプログラミング開発が得意で、PHP・JavaScript・HTML・CSS・MySQLの経験が長いです。
            </div>
          </div>
        </div>
        <div
          class="mb-6 p-4 md:w-1/3 md:mb-0 flex flex-col justify-center items-center max-w-sm mx-auto"
        >
          <div
            class="w-70 bg-white -mt-10 shadow-lg rounded-lg overflow-hidden p-5"
          >
            <div
              class="inline-flex"
            >
              <div
                class="flex-1 h-4 w-4 m rounded-full m-1 bg-yellow-100"
              >
                <div
                  class="h-2 w-2 rounded-full m-1 bg-yellow-500"
                />
              </div>
              <div
                class="text-gray-700"
              >
                ひとこと
              </div>
            </div>
            <div
              class="text-gray-600 text-left"
            >
              常に新しいこと・経験したことのないことにチャレンジしながら自分を成長していくことを心がけています。
            </div>
          </div>
        </div>
      </div>
    </div>
  </main>
  <footer>
    <p
      class="text-center text-gray-800"
    >
      © 2020 Yuuichi 
    </p>
  </footer>
</DocumentFragment>
`;
